pkg/webhook/validator.go

```diff
@@ -552,11 +561,11 @@ func ValidatePolicyAttestationsForAuthority(ctx context.Context, ref name.Refere
 			va, err := validAttestations(ctx, ref, verifier, rekorClient, remoteOpts...)
 			if err != nil {
 				logging.FromContext(ctx).Errorf("error validating attestations: %v", err)
-				return nil, fmt.Errorf("validating attestations: %w", err)
+				return nil, fmt.Errorf("key validation failed for authority %s for %s: %w", name, ref.Name(), err)
```

Comment:
looks like maybe copy&paste from signature validation, can you add something here about the attestation.

Reply:
I prefixed the signature paths with `signature` and the attestation paths with `attestation` to distinguish.  I guess I made them too similar 🤣